src/clients/storage/BlobStorageClient.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const request = require('request');
+const apiUrlBase = process.env.FORTIS_CENTRAL_ASSETS_HOST || 'https://fortiscentral.blob.core.windows.net';
+
+function buildTopicsSiteTypeUri(siteType) {
+  return `${apiUrlBase}/settings/siteTypes/${siteType}/topics/defaultTopics.json`;
+}
+
+/**
+ * @param {string} siteType
+ * @returns {Promise.<Array<{keyword: string, lang_code: string, insertion_time: timestamp, translations: Map<string, string>}>>}
+*/
+function fetchTopicsBySiteType(siteType) {
+  return new Promise((resolve, reject) => {
+    request.get(buildTopicsSiteTypeUri(siteType), (err, response, body) => {
+      if(err || response.statusCode !== 200) {
+        return reject(`Unable to get topics for siteType ${siteType}: ${err}`);
+      }
+
+      let topics;
+      try {
+        topics = JSON.parse(body);
+      } catch (err) {
+        return reject(`Unable to parse JSON for topics response ${body}: ${err}`);
+      }
+
+      resolve(topics);
+    });
+  });
+}
+
+module.exports = {
+  fetchTopicsBySiteType: fetchTopicsBySiteType
```

Comment:
Just as a general remark: it's a bit odd to have a client API that does something very specific which will be used only in one place. I'd normally expect client APIs to provide generic functionality (e.g. "insert into cassandra" or "fetch from blob") and then the business logic (i.e., in our code base the resolvers) to build on top of that (e.g. "fetch the topics for a site type from blob").

In the interest of time, this is fine by me for now, but consider to refactor this when you implement the resolver so that the blobStorageClient exposes a "fetchJson" method which takes a blob path and then the resolver takes care of formulating the topics-specific path.